src/herder/HerderSCPDriver.cpp

```diff
@@ -53,6 +54,10 @@ HerderSCPDriver::SCPMetrics::SCPMetrics(Application& app)
           app.getMetrics().NewCounter({"herder", "state", "current"}))
     , mHerderStateChanges(
           app.getMetrics().NewTimer({"herder", "state", "changes"}))
+    , mNominateToPrepare(
+          app.getMetrics().NewTimer({"scp", "value", "nominate"}))
```

Comment:
better names: `{"scp", "timing", "nominated" }` and `{"scp", "timing", "externalized"}`